index.js

```diff
@@ -1,12 +1,17 @@
 const express = require('express');
 const helmet = require('helmet');
 
+const zoosRouter = require('./zoos/zoos-router.js');
+const bearsRouter = require('./bears/bears-router.js');
```

Comment:
Good job separating out your routes, this is a much more realistic structure for when you work on bigger projects